testsuite/forge-cli/src/main.rs

```diff
@@ -1046,7 +1048,7 @@ fn realistic_env_workload_sweep_test() -> ForgeConfig {
         // Investigate/improve to make latency more predictable on different workloads
         criteria: [
             (5500, 0.35, 0.3, 0.8, 0.65),
-            (4500, 0.35, 0.3, 1.0, 1.0),
+            (4500, 0.35, 0.3, 1.0, 2.0), // At higher TPS, ordered_to_commit increases
```

Comment:
revert this one, I am changing this test much more

Reply:
Done, thanks @igor-aptos 😄